Main page body HTML markup and content images

Androborn opened the pull request:
*Adds Main page body HTML markup for:
- hero section
- gallery section
- more photo section

*Adds contentpictures and content images for:
- hero section
- gallery section
- more photo section

Androborn committed:
Main page body HTML markup and content images

*Adds Main page body HTML markup for:
- hero section
- gallery section
- more photo section

*Adds contentpictures and content images for:
- hero section
- gallery section
- more photo section

codajoker commented on src/partials/main/hero.html:
```diff
@@ -0,0 +1,733 @@
+  <div class="container">
+    <h1 class="page-title main__page-head">Інженерна, конструкторська та дизайнерська робота</h1>
+    <h2 class="section-title visually-hiden">Карусель зображень</h2>
+    <ul class="carousel">
```
Добавить класс `list` сброс стилей для списка

Androborn replied:
Добавил - будет в повторном коммите.

codajoker commented on src/partials/main/hero.html:
```diff
@@ -0,0 +1,733 @@
+          />
+          <img
+            src="./images/mobile/main/image.jpg"
+            alt="Фігурка робота, надрукована 3D принтером"
```
3D я бы обернул в lang "en" для лучшей семантики

Androborn replied:
Добавил только в заголовки - будет в повторном коммите.

*В атрибут alt span можно вписать только через [костыль](https://stackoverflow.com/questions/64719220/span-element-in-an-image-alt-attribute).
**Уточни этот момент у ментора, пож., стоит ли пихать span в alt?** 

*Где-то читал, что 3D входит в список часто употребляемых англицизмов (наряду с названиями именитых брендов), язык которых можно не уточнять для ридеров, но теперь не могу найти прув.
**Уточни этот момент у ментора, пож. - мне сугубо для себя.**

codajoker replied:
Я тоже слыша это тогда все ок я не увидел что там альт

codajoker commented on src/partials/main/hero.html:
```diff
@@ -0,0 +1,733 @@
+        <picture class="carousel__img carousel__img--round-top-right">
+          <source
+            media="(min-width: 1444px)"
+            srcset="./images/deckstope/main/image.webp 1x, ./images/deckstope/main/image@2x.webp 2x"
```
Путь к картинкам ../

Androborn replied:
Поправил - будет в повторном коммите.

Androborn committed:
Minor twiks to main page body html

- Adds class list for deault styles drop of hero, gallery, more photo sections
- Adds <span lang="en">3D</span> to h3 elements of gallery section
- images and SVG spriyte paths changed from ./images/... to ../src/images/...
